rust/persia-core/src/tensor.rs

```diff
@@ -79,10 +79,10 @@ impl DType {
             DType::I32 => (*&DLDataTypeCode::DLInt, 32),
             DType::I64 => (*&DLDataTypeCode::DLInt, 64),
             DType::U8 => (*&DLDataTypeCode::DLUInt, 8),
-            DType::U16 => (*&DLDataTypeCode::DLUInt, 16),
```

Comment:
don't map unsafely

Reply:
Already update the original unsigned int64 data to signed int64